cdap-ui/app/cdap/components/LogViewer/TopPanel.tsx

```diff
@@ -0,0 +1,193 @@
+/*
+ * Copyright © 2020 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import * as React from 'react';
+import withStyles, { WithStyles, StyleRules } from '@material-ui/core/styles/withStyles';
+import DataFetcher from 'components/LogViewer/DataFetcher';
+import Button from '@material-ui/core/Button';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import Checkbox from '@material-ui/core/Checkbox';
+import classnames from 'classnames';
+import If from 'components/If';
+import IconButton from '@material-ui/core/IconButton';
+import Close from '@material-ui/icons/Close';
+
+export const TOP_PANEL_HEIGHT = '50px';
+
+const styles = (theme): StyleRules => {
+  return {
+    root: {
+      backgroundColor: theme.palette.grey[100],
+      color: theme.palette.white[50],
+      display: 'flex',
+      justifyContent: 'flex-end',
+      alignItems: 'center',
+      height: TOP_PANEL_HEIGHT,
+      paddingLeft: '20px',
+      paddingRight: '20px',
+      position: 'relative',
+    },
+    actionButton: {
+      margin: theme.spacing(1),
+
+      '&:hover': {
+        color: theme.palette.white[50],
+        borderColor: theme.palette.white[50],
+        backgroundColor: theme.palette.grey[200],
+      },
+
+      '&$disabled': {
+        // needed to beat specificity
+        color: theme.palette.grey[50],
+        cursor: 'not-allowed',
+        backgroundColor: theme.palette.white[50],
+      },
+    },
+    disabled: {},
+    checkboxContainer: {
+      margin: '0 40px',
+      userSelect: 'none',
+    },
+    checkbox: {
+      color: theme.palette.white[50],
+    },
+    error: {
```

Comment:
Any specific reason why we are introducing a new error pattern? Can we reuse the `Alert` component? This way when we replace it with material alerts tomorrow we would be able identify all the usages of the errors and replace them for consistency.